src/Repository/EventRepository.php

```diff
@@ -512,6 +512,20 @@ public function countReferentEventsInReferentManagedArea(Adherent $referent): ar
         return $this->aggregateCountByMonth($result, BaseEvent::REFERENT_EVENT_TYPE.'s');
     }
 
+    public function countTotalEventsInReferentManagedAreaForCurrentMonth(Adherent $referent): int
+    {
+        $this->checkReferent($referent);
+
+        $query = $this->queryCountByMonth($referent, 1);
```

Comment:
with this it will be `count` of the previous month + current month, and not only of current month, no?

Reply:
Surely! Thanks !
Fixed.

Reply:
Sure :)?
https://3v4l.org/Hmi82